bindings/pyroot_experimental/pythonizations/python/ROOT/_application.py

```diff
@@ -52,6 +52,9 @@ def _inputhook_config():
         # PyOS_InputHook-based mechanism
         # Point to a function which will be called when Python's interpreter prompt
         # is about to become idle and wait for user input from the terminal
+        # We have to import the readline module, which starts calling the PyOS_InputHook
+        # also if Python is in script mode and not in interactive mode.
+        import readline
```

Comment:
Thanks @stwunsch ! Why does the activation of the interactive mode need to be here, though? I would rather say we should have it in `rootbrowse`. In the case of inspection of ROOT graphics from the Python prompt, we will already have the interactivity enabled.

Reply:
But then every user who uses a Python script as driver for ROOT graphics has to know this. This does not seem feasible to me. Also it's not backwards compatible.